### ISSUE TITLE
Sprint 41 (2/29/24-3/13/24)

### ISSUE THREAD

**Issue by lmorris3** [department-of-veterans-affairs/va.gov-team#77242]
## PTO

2/29-3/4 - Lauren OOO
3/4-3/5 - Anu OOO
3/4-3/8 - Mateo OOO
3/6-3/8 - Carl OOO
3/8 - Phil OOO starting at 12pm ET

## Sprint 41 Goals

**Campaigns**
_High-level goal: Launch behavior change communications campaign._ 

Engineering:
- USiP PR, launch (blocked until SSiT OCTO gives word)

Design/Research:
- Deploy GovDelivery segmented emails (blocked, may unblock this sprint)

Data:
- Monitor sign in and call center data (blocked until campaigns resume)
- Automate MPI, SSOe, MHV data ingest
- finish source account creation data view
- Pull monthly dataset for Login.gov and ID.me account creation and use (send to Tom 1st of the month)

Product:
- Identity roadmap with Tom

**Stretch Goals**

**IPP**
_High level goal: Prepare for IPP Pilot_

Engineering:
- Dockerize application to spin up next.js, DB instance (@moanu404 is working on this and will pair with @rdvyas)
- IPP Security Counter-Measures Priority 10 (@rdyvas will be prioritizing this)
- Proofing Agent - Case number lookup (almost ready for review - @moanu404 needs to write a description)
- Proofing Agent - Update Case number status to proofed (tie record and change status to proofed. small integration.)
- IdP - Sinatra proxy auth server (merged just need to set up for sign in pages - covered in following tickets)
- Develop a sign in page for Veterans 
- Develop a sign in page for Proofing Agents
     - Update authentication request to require proofing agent to sign in with PIV (requirement of line above)
     - Extract x.509 from authentication response
- Onboarding to VA Datadog (for Logging, monitoring, observability) - Steve
-  Timeline artifact (realistic timeline plot technical tasks and overlay with pilot plans @SophiaPhilipMO @rdvyas will draft timeline and walk through with @lmorris3)

Research/Design:
- Pilot planning: finalize pilot site
- Conduct Proofing Agent Usability Tests
- SME interview synthesis
- Pilot onsite signage (mockup but don't need to finalize)
- Complete IPP research plan (drafted and in review QA - need to formally submit a research ticket; create 3 tickets for first pilot)

Product:
- Draft DIRA Statement (DIRA not needed bc MPI is authoritative source), review with Tom
- revise IPP charter to show vision of an in-person proofing service with VA

**Stretch Goals**
- Proofing Agent Usability Test synthesis (we can start this, create template)
- Design A11y improvements to Veteran app (check in with @lmorris3 - move to dev design sync)
- SME interview research report (start and might be able to get this in a solid draft)
- Data wipeout plan for pilot
- Deploy intake app to LHDI QA env 
- Complete lightkeeper/k8s setup (want whole team to be enabled on this for pilot)
- VA Notify Integration Discovery and Dev start (for 1st pilot we may not need this and we move this to a following integration; see who is the main POC needed to maintain throughout all of the pilots. there is a discovery ticket on this #74319. **get a name by this sprint**. identify stakeholders and begin to maintain a doc)

## What we want to demo at the end of the sprint

- IPP Veteran Intake sign in screen
- IPP Proofing Agent sign in screen
- LHDI deployment
- SME Interview Insights
